CalibCalorimetry/EcalTPGTools/plugins/EcalTPGDBApp.cc

```diff
@@ -245,24 +250,24 @@ int EcalTPGDBApp::writeToConfDB_TPGLUT(const map<EcalLogicID, FEConfigLUTGroupDa
 
   result = fe_lut_info.getId();
 
-  cout << "*****************************************" << endl;
-  cout << "************LUT done*********************" << endl;
-  cout << "*****************************************" << endl;
+  edm::LogVerbatim("ECALTPGDBApp") << "*****************************************";
+  edm::LogVerbatim("ECALTPGDBApp") << "************LUT done*********************";
+  edm::LogVerbatim("ECALTPGDBApp") << "*****************************************";
   return result;
 }
 
 int EcalTPGDBApp::writeToConfDB_TPGWeight(const map<EcalLogicID, FEConfigWeightGroupDat>& lutgroupset,
                                           const map<EcalLogicID, FEConfigWeightDat>& lutset,
                                           int ngr,
                                           string tag) {
-  cout << "*****************************************" << endl;
-  cout << "************Inserting weights************" << endl;
-  cout << "*****************************************" << endl;
+  edm::LogVerbatim("ECALTPGDBApp") << "*****************************************";
+  edm::LogVerbatim("ECALTPGDBApp") << "************Inserting weights************";
+  edm::LogVerbatim("ECALTPGDBApp") << "*****************************************";
 
   int result = 0;
 
   FEConfigWeightInfo fe_wei_info;
-  fe_wei_info.setNumberOfGroups(5);  // this eventually refers to some other table
+  fe_wei_info.setNumberOfGroups(ngr);  // this eventually refers to some other table
```

Comment:
Is this comment still needed, now that the parameter `ngr` is passed by the module, and not statically set to "5"?